.github/workflows/main.yml

```diff
@@ -10,9 +10,9 @@ jobs:
       matrix:
         node-version: [16.x]
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
```

Comment:
Are these github action upgrades needed for the tests to run now? Did you think about making a smaller PR with just these github actions upgrades in them? My apologies if you did and there is some really good reason for this to be in the PR. Anyways keep them for now and thanks for upgrading this stuff.

Reply:
Might have been better to shoot for main and then rebase main with this. anyways thanks for the upgrade.